plugins/hls-class-plugin/src/Ide/Plugin/Class/CodeAction.hs

```diff
@@ -113,30 +115,31 @@ codeAction recorder state plId (CodeActionParams _ _ docId _ context) = pluginRe
             logWith recorder Info (LogImplementedMethods cls implemented)
             pure
                 $ concatMap mkAction
-                $ fmap (filter (\(bind, _) -> bind `notElem` implemented))
-                $ minDefToMethodGroups range sigs
-                $ classMinimalDef cls
+                $ nubOrdOn snd
```

Comment:
Is this for `Action is skipped when only required methods are missing to avoid duplication`?

Can you add a test for this?

Reply:
I did add test for that :wink: Maybe naming is a bit unfortunate? :thinking: 

[`Add placeholders for all missing methods is unavailable when all methods are required`](https://github.com/batkot/haskell-language-server/blob/a704dfe21329d6d555988e4121f4db2fd8f464cd/plugins/hls-class-plugin/test/Main.hs#L73-L76)